sources.cfg

```diff
@@ -6,3 +6,8 @@ extensions +=
     mr.developer
 
 auto-checkout =
+    ftw.testing
+    ftw.builder
+
+[branches]
+ftw.builder = python3
```

Comment:
🚧 https://github.com/4teamwork/ftw.builder/pull/59 needs to be merged / released first.

Reply:
Using now ftw.builder from master.